examples/random-dog/dog-image-widget/src/widget/index.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+    WidgetOptionItemType,
+    init,
+    EventType,
+} from "@datadog/ui-extensions-sdk";
+import React, { useEffect, useState } from "react";
+import ReactDOM from "react-dom";
+
+import "./widget.css";
+import "./../index.css";
+import "typeface-roboto";
+import "milligram";
+
+const client = init();
+
+function Widget() {
+    const [ breeds, setBreeds ] = useState([])
+    const [ image, setImage ] = useState(null);
+
+    useEffect(() => {
+            fetch("http://localhost:3001/breeds")
+                .then(res => res.json())
+                .then(({ breeds }) => setBreeds(breeds))
+                .catch(err => console.log('An error occurs on fetching breeds', err))
+    }, [])
+
+    useEffect(() => {
+            if (breeds.length) {
+                const options = breeds.map(breed => ({
+                    label: breed['name'],
+                    value: breed['id']
+                }))
```

Comment:
Add `All Breeds` option to widget (already supported by api)
```suggestion
                var options = breeds.map(breed => ({
                    label: breed['name'] as string,
                    value: breed['id'] as string
                }))
                options.unshift({ label: "All Breeds", value: "0" })
```